### PR TITLE
{devel}[GCCcore/9.3.0] libelf v0.8.13

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/c8e38d45e9f33b4f8ac9079a4a2255fa for a full test report.

File: easybuild/easyconfigs/l/libelf/libelf-0.8.13-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libelf'
+version = '0.8.13'
+
+homepage = 'http://www.mr511.de/software/english.html'
```
Comment (verdurin) [inline easybuild/easyconfigs/l/libelf/libelf-0.8.13-GCCcore-9.3.0.eb:6]: @migueldiascosta wondering whether we can do anything about this homepage, which gives a 404 for me with http and a security error with https...

Comment (lexming) [reply easybuild/easyconfigs/l/libelf/libelf-0.8.13-GCCcore-9.3.0.eb:6]: We could use the page in the directory of `fsf.org`: https://directory.fsf.org/wiki/Libelf

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node321.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/974ffdbf4601da3a29676b5571e96821 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @migueldiascosta !

**Pull request merged**